lib/ueberauth/strategy/facebook.ex

```diff
@@ -181,11 +175,11 @@ defmodule Ueberauth.Strategy.Facebook do
   end
 
   defp option(conn, key) do
-    default = Keyword.get(default_options(), key)
+    default = Dict.get(default_options, key)
```

Comment:
Dict isn't going to work here :( Dict is deprecated. When is default_options anything other than a keyword?